jdaviz/configs/mosviz/mixins.py

```diff
@@ -0,0 +1,56 @@
+__all__ = ['Matched2dSpectrumMixin']
```

Comment:
Since you are putting this in API doc, should have a module level docstring to explain the purpose of this *module*.